exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java

```diff
@@ -29,141 +35,131 @@ public  Game(){
 	public String createRockQuestion(int index){
 		return "Rock Question " + index;
 	}
-	
+
 	public boolean isPlayable() {
 		return (howManyPlayers() >= 2);
 	}
 
-	public boolean add(String playerName) {
-		
-		
+	public boolean addPlayer(String playerName) {
 	    players.add(playerName);
-	    places[howManyPlayers()] = 0;
-	    purses[howManyPlayers()] = 0;
+	    playerPositions[howManyPlayers()] = 0;
+	    playerPurses[howManyPlayers()] = 0;
 	    inPenaltyBox[howManyPlayers()] = false;
-	    
+
 	    System.out.println(playerName + " was added");
 	    System.out.println("They are player number " + players.size());
 		return true;
 	}
-	
+
 	public int howManyPlayers() {
 		return players.size();
 	}
 
 	public void roll(int roll) {
 		System.out.println(players.get(currentPlayer) + " is the current player");
 		System.out.println("They have rolled a " + roll);
-		
+
 		if (inPenaltyBox[currentPlayer]) {
 			if (roll % 2 != 0) {

```

Comment:
How would you go about removing the nested conditionals? Would you simply place it in one if, or do some cool polymorphism? Just curious